src/model/page.urls.ts

```diff
@@ -1,5 +1,10 @@
-export const ROOT: string = "/";
-export const PROMISE_TO_FILE: string = "/promise-to-file";
-export const COMPANY_NUMBER: string = "/company-number";
+import * as TEMPLATE from "./template.paths";
+
+/**
+ * Keep template names in template.paths.ts and reference them in here for URLs
+ */
 
-export const PROMISE_TO_FILE_COMPANY_NUMBER: string = PROMISE_TO_FILE + COMPANY_NUMBER;
+const SEPARATOR: string = "/";
+export const ROOT: string = "/";
+export const PROMISE_TO_FILE: string = SEPARATOR + "promise-to-file";
+export const COMPANY_NUMBER: string = SEPARATOR + TEMPLATE.COMPANY_NUMBER;
```

Comment:
This company number page URL looks wrong - there's no preceding '/promise-to-file' prefix.

Reply:
Added some comments to the file to explain that the urls extend a base url set in the app.